src/ripple/app/ledger/impl/LedgerMaster.cpp

```diff
@@ -292,8 +295,9 @@ class LedgerMasterImp
         {
             // Calculate the sample median
             std::sort (times.begin (), times.end ());
-            signTime = (times[times.size() / 2] +
-                times[(times.size() - 1) / 2]) / 2;
+            auto const t0 = times[(times.size() - 1) / 2];
+            auto const t1 = times[times.size() / 2];
+            signTime = t0 + (t1 - t0)/2;
```

Comment:
Any particular reason for changing the equation? I don't mind, and the result will still be the same (which maybe not be obvious at first).